### PR TITLE
Implemented ValueTransfer class

### PR DESCRIPTION
**Pull request by jimni1222**
## Proposed changes

This PR introduces implementation of ValueTransfer class.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-js/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-js)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

refer #249 

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/caver-transaction/src/transactionTypes/valueTransfer/valueTransfer.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+     */
+    static decode(rlpEncoded) {
+        return new ValueTransfer(_decode(rlpEncoded))
+    }
+
+    /**
+     * Creates a value transfer transaction.
+     * @constructor
+     * @param {object} createTxObj - The parameters to create a ValueTransfer transaction.
+     *                               The createTxObj can define `from`, `to`, `value`, `nonce`, `gas`, `gasPrice`, `signatures` and `chainId`.
```
Comment (kjhman21) [inline packages/caver-transaction/src/transactionTypes/valueTransfer/valueTransfer.js:64]: This comment does not describe that createTxObj can be an RLP-encoded string.

Comment (jimni1222) [reply packages/caver-transaction/src/transactionTypes/valueTransfer/valueTransfer.js]: I added description :)

File: packages/caver-transaction/src/transactionTypes/valueTransfer/valueTransfer.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        to,
+        value: utils.trimLeadingZero(value),
+        from,
+        signatures,
+    }
+}
+
+/**
+ * Represents a value transfer transaction.
+ * Please refer https://docs.klaytn.com/klaytn/design/transactions/basic#txtypevaluetransfer to see more detail.
```
Comment (kjhman21) [inline packages/caver-transaction/src/transactionTypes/valueTransfer/valueTransfer.js:46]:
```suggestion
 * Please refer to https://docs.klaytn.com/klaytn/design/transactions/basic#txtypevaluetransfer to see more detail.
```

File: packages/caver-transaction/src/transactionTypes/valueTransfer/valueTransfer.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    get value() {
+        return this._value
+    }
+
+    set value(val) {
+        this._value = utils.numberToHex(val)
+    }
+
+    /**
+     * Returns RLP-encoded transaction string(rawTransaction).
```
Comment (kjhman21) [inline packages/caver-transaction/src/transactionTypes/valueTransfer/valueTransfer.js:112]:
```suggestion
     * Returns the RLP-encoded string of this transaction (i.e., rawTransaction).
```

File: packages/caver-transaction/src/transactionTypes/valueTransfer/valueTransfer.js [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+                this.to,
+                Bytes.fromNat(this.value),
+                this.from,
+                this.signatures,
+            ]).slice(2)
+        )
+    }
+
+    /**
+     * Returns common RLP-encoded transaction string for making signature.
```
Comment (kjhman21) [inline packages/caver-transaction/src/transactionTypes/valueTransfer/valueTransfer.js:133]:
```suggestion
     * Returns the RLP-encoded string to make the signature of this transaction.
```

**Pull request merged**